docs/01_gettingStarted/03_developer/02_typescript.md

```diff
@@ -94,14 +94,14 @@ In PaddleSystem.ts we see a good example of this reactive state pattern. The app
 ```
 export const PaddleState = defineState({
   name: 'ee.pong.PaddleState',
-  initial: {} as Record<
+  initial: {} as Record
```

Comment:
revert this, the the `<` and `>` are required by typescript

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
Turns out code blocks do allow <> - it was somewhere else that this needed to be escaped... so this is now corrected.